modules/client-configuration/pages/clients-microos.adoc

```diff
@@ -0,0 +1,144 @@
+[[clients-microos]]
+= Registering {opensuse}{nbsp}{microos} Clients
+
+This section contains information about registering clients running these {opensuse}{nbsp}{microos} operating systems:
+
+* {opensuse}{nbsp}{microos}
+
+
+
+[WARNING]
+====
+This section is provided as a technology preview for testing purposes.
+Do not use this feature on production systems.
+====
+
+
+
+== Add Software Channels
+
+Before you register {opensuse}{nbsp}{microos} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+The channels you need for this procedure are:
+
+[[microos-channels-cli]]
+[cols="1,1,1,1", options="header"]
+.MicroOS Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Updates Channel
+
+| {opensuse}{nbsp}{microos}
+| opensuse_tumbleweed
+| opensuse_tumbleweed-non-oss
+| opensuse_tumbleweed-update
+
+|===
+
+include::snippets/addchannels_novendor_cli.adoc[]
+
+
+
+== Check Synchronization Status
+
+ifeval::[{suma-content} == true]
+
+include::snippets/check_sync_webui_suma.adoc[]
+
+endif::[]
+
+ifeval::[{uyuni-content} == true]
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+endif::[]
+
+include::snippets/check_sync_cli.adoc[]
+
+[IMPORTANT]
+====
+{opensuse}{nbsp}{microos} channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+
+
+ifeval::[{uyuni-content} == true]
```

Comment:
Why only for Uyuni?

Reply:
Hold over from when it was in the previous chapter.